components/bsx/Create/CreateCollectionParent.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
TBH the component `CreateCollectionParent` seems a little bit strange : )

Reply:
what would you rather have it named? createcollection was already there, I couldnt figure out anything better :D

Reply:
CreateCollectionPage / CreateCollectionForm

Reply:
changed, thanks